src/styles/global.css

```
@@ -36,7 +36,8 @@ html {
 }
 
 body {
-	font-family: "Inter", "Shippori Mincho", "Noto Serif JP", system-ui, sans-serif;
+	font-family:
+		"Inter", "Shippori Mincho", "Noto Serif JP", system-ui, sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)